101-application-gateway-rewrite/azuredeploy.parameters.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "backendIpAddress1": {
+      "value": "GEN-CUSTOM-FQDN-NAME"
```

Comment:
this does not generate an IP address, just remove this and use your defaultValue